README.md

```diff
@@ -29,22 +32,68 @@ Scripts Setup
    e.g. for Ubuntu/Debian
 
        sudo apt-get install cdo nco netcdf-bin
-       
+
 5. Adjust the settings.config to your liking
 
 Usage
-=====
+-----
 1. After completing both setups, you can use this script as follows:
 
-       ./bin/get_data.sh <date> <time>
+       ./bin/get_cds.sh <date> <time>
 
    For example, to get ERA5 data for March 2nd 2020 at 12 o'clock, use
 
-       ./bin/get_data.sh 2020-03-02 12:00:00
+       ./bin/get_cds.sh 2020-03-02 12:00:00
 
 2. Done, copy the .nc files to your mss data directory and give them their appropriate suffix.\
-   Using the demodata for MSS, this is ~/mss/testdata and EUR_LL015 suffix. 
+   Using the demodata for MSS, this is ~/mss/testdata and EUR_LL015 suffix.
 
        for file in ./mss/*.nc; do mv "$file" "${file/.nc/.EUR_LL015.nc}"; done
        mv ./mss/*.nc ~/mss/testdata
 
+
+ECMWF forecast
+==============
+
+ECTrans Setup
+-------------
+1. Login at https://ecaccess.ecmwf.int/ecmwf/ \
+   Go to https://ecaccess.ecmwf.int/ecmwf/gateway/ECtrans/Setup
+2. Click "Add association" at the bottom of the page \
+   Call the association "MSS-Data-Transfer" and set up your SFTP/FTP Server to your liking \
+   If you want to call it something else, make sure to change `ectrans_id` inside the `settings.config`
```

Comment:
This would not work. ectrans_id is explicitely set in get_ecmwf.sh (to my personal one). 
Needs to be updated such that ectrans_id is imported.

Reply:
Same for area and grid setup. This should be read from settings.config in get_ecmwf.sh

Reply:
Correct, the ecmwf part does not import that. We should define - in a second step, where to best place the configuration such that the script will be able to find it.